avalon/tools/projectmanager/widget.py

```diff
@@ -165,8 +165,8 @@ class AssetModel(TreeModel):
     Deprecated = 2
     ObjectId = 3
 
-    DocumentRole = QtCore.Qt.UserRole + 1
-    ObjectIdRole = QtCore.Qt.UserRole + 2
+    DocumentRole = QtCore.Qt.UserRole + 2
```

Comment:
To describe why this has changed; This fixes it overriding the NodeRole as it did previously.